utils/saasdial/auth.go

```diff
@@ -0,0 +1,44 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package saasdial
+
+import (
+	"context"
+
+	"google.golang.org/grpc/credentials"
+)
+
+type platformAuth struct {
+	sessionID string
+}
+
+// GetRequestMetadata implements credentials.PerRPCCredentials interface.
+func (b *platformAuth) GetRequestMetadata(ctx context.Context, uri ...string) (map[string]string, error) {
+	return map[string]string{
+		"authorization": platformAuthType + " " + b.sessionID,
+	}, nil
+}
+
+// RequireTransportSecurity implements credentials.PerRPCCredentials interface.
+func (*platformAuth) RequireTransportSecurity() bool {
+	return false
```

Comment:
⚠️ @artemgavrilov I think that one should be `true`, not `false` – we don't want to send authentication information over insecure connection. Please change it and test if it works.

Reply:
Fixed.

I've tested local setup, everything works

Reply:
Ahhh... no, I tested that everything works with our dev. Will test also local infra setup